fvm/evm/stdlib/contract.cdc

```diff
@@ -815,4 +813,19 @@ contract EVM {
             ?.borrowBridgeAccessor()
             ?? panic("Could not borrow reference to the EVM bridge")
     }
+
+    /// The Heartbeat resource controls the block production 
+    access(all) resource Heartbeat {
+
+        /// heartbeat calls commit block proposals and forms new blocks including all the 
+        /// recently executed transactions.
+        /// The Flow protocol makes sure to call this function once per block as a system call. 
+        access(all) fun heartbeat() {
```

Comment:
Given the heartbeat resource will be stored in account storage, and only the node software has access to it, using `access(all)` should be fine here 👍